### PR TITLE
docs: 📚 supernumber pattern - add information/guidance

### LINKED ISSUE #961: docs: 📚 supernumber pattern - add information/guidance
## Description

[Request from 16.04.2024
](https://teams.microsoft.com/l/message/19:sycqZKo6K_-P5eYh0H4Xn2UpoKmti8Ar5SICMb9z7C41@thread.tacv2/1713276435713?tenantId=2eb07b91-f9f8-437b-a89e-59f55e565c11&groupId=cec93f98-ceee-42bd-bfde-a6b51fc1194e&parentMessageId=1713276435713&teamName=Solid%20Design%20System%20Community%20(Team)&channelName=General&createdTime=1713276435713)

Missing the following information:

The supernumber is only displayed in one variant (all text elements enabled). In the old component the text above and below the number were optional so it would be nice to have this reflected in the design as well (for example in case of different spacings).

In the design the supernumber has 3 sizes (lg, md, sm) can you give guidance when to use which size for example:

Use sm when the component is smaller than 300px
Use md when the component is smaller than 400px
….

Since the component is called supernumber are you still allowed to place normal text inside the highlighted area or for example only a number + unit?

--

https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/docs/pattern-supernumber--docs

```
Default
Used to highlight and convey important key figures. The component consists of overline, number, subline and description. Supernumber is used as a subcomponent inside sd-container.
```

- [ ] Refactor Storybook descriptions: 
  - [x] This is a pattern - replace all "component" phrases with "pattern"
  - [x] Specify which parts are optional (overline, subline, description) and that prefix / suffix are static. Specify - use prefix/suffix option of countup.js ... sizes are examples ...
  - [x] make clear, that we just provide samples how to use the countUp plugin (eg: "Use the `prefix` option to add a prefix to the supernumber." => "If you need a prefix, you can use the `prefix` option from countUp.")
- [x] To do Figma: Specify which parts are optional (overline, subline, description)
- [x] To do Figma: Explain sizes
- [x] To do Figma: Add prefix / suffix info about animation
- [x] Storybook bug - small mobile overline/subline need to be centered

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

### PR DESCRIPTION
**Pull request by Vahid1919**
## Description:

 Improved supernumber pattern documentation and removed "component" terminology in all patterns that included it. Closes #961.

## Definition of Reviewable:

- [x] Documentation is created/updated
- [x] Stories (features, a11y) are created/updated
- [x] relevant tickets are linked

### CODE REVIEW

**Review by karlbaumhauer — APPROVED** [review]

**Review by yoezlem — APPROVED** [review]

**Review by azraefendic — APPROVED** [review]

**Comment by Vahid1919** [comment]
@coraliefeil here is the chromatic link for easy access: https://64469bd84c5ea1ce2dd5d016-rcizkxtrkn.chromatic.com/?path=/docs/pattern-supernumber--docs

File: packages/components/src/patterns/supernumber/supernumber.stories.ts [hunk 1]
```diff
@@ -14,14 +14,76 @@ export default {
...
+  render: () => html`
+    <div class="sd-container sd-container--variant-white flex flex-col items-center text-center">
+      <p class="sd-paragraph font-bold mb-2">Nisi eu excepteur anim esse</p>
+      <div class="text-[72px] text-primary leading-[86.4px]">XXX</div>
+    </div>
+  `
+};
+
+/**
+ * The supernumber can be displayed with a Subline. The subline is displayed below the number.
```
Comment (coraliefeil) [inline packages/components/src/patterns/supernumber/supernumber.stories.ts:62]: * The supernumber can be displayed with a subline. The subline is displayed below the number.

**Comment by coraliefeil** [comment]
just 1 little typo .. then it’s good. thx

**Review by mariohamann — APPROVED** [review]

File: packages/components/src/patterns/supernumber/supernumber.stories.ts (hunk 1 above)
Comment (Vahid1919) [inline packages/components/src/patterns/supernumber/supernumber.stories.ts:62]:
```suggestion
 * The supernumber can be displayed with a subline. The subline is displayed below the number.
```

**Pull request merged**